src/documents/ViewAPI.md

```diff
@@ -0,0 +1,174 @@
+* Returns: `{Component}` An Infusion [view component](ComponentConfigurationOptions.md#view-components) managing the interaction with the label. ***Note:*** disposing of this component is the responsibility of the caller. If you have destroyed
+the markup to which this component is attached, you must destroy the component using its `destroy()` method or it will leak. Successive calls to `fluid.updateAriaLabel` for the same DOM node will return the same component. 
+
+#### Structure of `AriaLabellerOptions`
+ 
+| Name | Type | Description | Default |
+|------|------|-------------|---------|
+|`dynamicLabel`| `Boolean`| Whether or not an ARIA  live region should be created an associated with the element, as well as the use of `aria-label`| `false` |
+|`liveRegionMarkup`| `String` | The markup to use to create the live region (only used if `dynamicLabel` is `true`).|`"<div class=\"liveRegion fl-offScreen-hidden\" aria-live=\"polite\"></div>"`|
+|`liveRegionId`| `String` | The ID to assign to the live region (only used if `dynamicLabel` is true). |`"fluid-ariaLabeller-liveRegion"`|
+|text| `String` | The text to place in the live region (only used if `dynamicLabel` is true).|none|
+
+Note that as well as forming the 3rd argument to `fluid.updateAriaLabel`, these options also form the [component options](ComponentOptionsAndDefaults.md) to the associated component, which is 
+an Infusion `fluid.viewComponent` of type `fluid.ariaLabeller`.
+
+## Managing Focus and Blur
+
+Infusion includes two "major utilities" for managing focus interactions, [`fluid.globalDismissal`](#fluid-globaldismissal-nodes-dismissfunc-) and [`fluid.deadMansBlur`](#fluid-deadmansblur-control-options-) for managing focus interactions, as well as a few minor utilities.
+
+### fluid.globalDismissal(nodes[, dismissFunc])
+
+* `nodes {Object: String  → DomElement|jQuery}` A free hash of names onto nodes that a click ***outside*** of will trigger `dismissFunc`
+* `dismissFunc {Function: DomEvent  →  None}|Undefined` A function which will be executed when a "dismissal click" is received in the document. If this argument is not supplied, any existing dismissal function will be disarmed.   
+    
+Used in the stereotypical situation where a click event (morally, one which shifts focus outside a dialog or other region of interaction - but only actual `click` events
+are supported by this function) should have the effect of "dismissing` the dialog or interaction region. This accepts a free hash of nodes designating the interaction
+region and an optional "dismissal function". If `dismissFunc` is set, this "arms" the dismissal system, such that when a click
+is received OUTSIDE any of the DOM node hierarchy covered by `nodes`, the dismissal function will be executed, and supplied with the DOM event
+triggered by the click event. `fluid.globalDismissal` is a `one shot deal` - if the `dismissFunc` is executed, the system is disarmed and will need
```

Comment:
The quotes around "one shot deal" should probably be double quotes (single quotes result in code-styling).